src/doc/src/faq.md

```diff
@@ -294,3 +294,16 @@ causes and provide diagnostic techniques to help you out there:
 [conventions in place]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#-sys-packages
 [`direct-minimal-versions`]: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#direct-minimal-versions
 [custom merge tool]: https://github.com/rust-lang/cargo/issues/1818
+
+## Why does my build take up so much space?
+
+Cargo trades off disk space for faster builds including:
```

Comment:
Do we want to put some tracking issue for reference here (if people are interested)?

Reply:
Any you have in mind?

The only one I can think of that would be relevant is #5026

Reply:
Or <https://github.com/rust-lang/cargo/issues/13136> or <https://github.com/rust-lang/cargo/issues/12633> but they all looks a bit far away. Anyway, we can add later when we have a more dedicated one.